src/cc/bcc_syms.cc

```diff
@@ -282,13 +295,23 @@ void ProcSyms::Module::load_sym_table() {
 }
 
 bool ProcSyms::Module::contains(uint64_t addr, uint64_t &offset) const {
-  for (const auto &range : ranges_)
+  for (const auto &range : ranges_) {
     if (addr >= range.start && addr < range.end) {
-      offset = (type_ == ModuleType::SO || type_ == ModuleType::VDSO)
-                   ? (addr - range.start + range.file_offset)
-                   : addr;
+      if (type_ == ModuleType::SO || type_ == ModuleType::VDSO) {
+        // Offset within the mmap
+        offset = addr - range.start + range.file_offset;
+
+        // Offset within the ELF for SO symbol lookup
+        if (type_ == ModuleType::SO)
```

Comment:
This condition is redundant since for for ModuleType::VDSO, `elf_so_addr_ - elf_so_offset_` should be 0 already.

Reply:
Ok. I'll get rid of it